src/client/components/side-navigation/sidebar.stories.js

```diff
@@ -10,16 +9,6 @@ export default {
   decorators: [withKnobs],
 };
 
-const options = {
-  active: true,
-  inactive: false,
-};
-
 export const SidebarActive = () => {
-  const text = ('value', 'dashboard');
-  const isActive = boolean('active', false, options);
-  const handleClick = action('clicked');
-  return (
-    <SidebarMenu text={text} isActive={isActive} handleClick={handleClick} />
-  );
+  return <SidebarMenu />;
```

Comment:
You need to supply the required props (that you removed in proptypes) for this to work in storybook. It's broken now.

The stuff you removed above looks to have been correct.

Reply:
While working with `side-navigation` component, we have `navigationButton`component which has the same attributes isActive & handleClick.We also released that the 4 buttons in `sidebar-navigation` have different functionalities and, if we are giving these attributes to `sidebar-navigation` all the buttons take in same functionality and that should n't happen and as the `navigation-button` has the isActive and handleclick, we are using it in that component.